pkg/kubelet/server/auth.go

```diff
@@ -99,7 +101,8 @@ func (n nodeAuthorizerAttributesGetter) GetRequestAttributes(u user.Info, r *htt
 			subresources = append(subresources, "pods")
 		case isSubpath(requestPath, healthz.DefaultHealthzPath):
 			subresources = append(subresources, "healthz")
-		case isSubpath(requestPath, configz.DefaultConfigzPath):
+		case isSubpath(requestPath, configz.DefaultConfigzPath),
+			isSubpath(requestPath, flagz.DefaultFlagzPath):
```

Comment:
move to the lower switch statement, out of the `KubeletFineGrainedAuthz` block

Reply:
Done, thx.

Reply:
> move to the lower switch statement, out of the `KubeletFineGrainedAuthz` block

@tallclair Currently, `configz` is inside `KubeletFineGrainedAuthz`. Do we need to move `flagz` outside of `KubeletFineGrainedAuthz`?

Reply:
That's fine. The endpoints in this switch statement have the `proxy` subresource appended for backwards compatibility. Since `flagz` is a new endpoint, it doesn't need to support the proxy permission.